test/e2e/utils_test.go

```diff
@@ -730,6 +753,163 @@ func cleanupCRP(name string) {
 	Eventually(removedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove CRP %s", name)
 }
 
+// cleanupCRO deletes the CRO and waits until the resources are not found.
+func cleanupCRO(number int) {
```

Comment:
```suggestion
func cleanupClusterResourceOverrides(number int) {
```

shortname is better to be used in the smaller scope, eg, variable name.